Integrations/integration-Alexa.yml

```diff
@@ -0,0 +1,119 @@
+description: Alexa provides website ranking information that can be useful in determining
+  if the domain in question has a strong web presence.
+detaileddescription: To use this integration, simply call ```!awis-info```. !awis
+  accepts "url" as a parameter to query.
+configuration:
+- display: Sensitivity threshold for configuring which domains are suspicious versus
+    trusted.
+  name: threshold
+  defaultvalue: "2000000"
+  type: 0
+  required: true
+script:
+  script: |
+    import urllib, sys, re
+    import xml.etree.ElementTree as ET
+    from collections import defaultdict
+
+
+    """GLOBAL VARIABLES/CONSTANTS"""
+    THRESHOLD = int(demisto.params().get('threshold'))
+
+
+    """MAIN FUNCTIONS"""
+    def awis(domain):
+        xml = urllib.urlopen('http://data.alexa.com/data?cli=10&dat=s&url=%s'%domain).read()
+        root = ET.fromstring(xml)
+        return root
+
+
+    """COMMAND FUNCTIONS"""
+    def awis_command():
+        domain = demisto.args().get('domain')
+        root = awis(domain)
+        dbot_ec = defaultdict()
+        dom_ec = defaultdict()
+        # Wrapping in Try Except since when no results are found, endpoint returns 404
```

Comment:
Can't we handle that from the response of the API?

Reply:
The issue is that it really isn't an API as much as it is an xml page. When the domain is searched for and not found, it returns a page with nothing on it. So when we attempt to parse the xml, it crashes out.